ci/recipe/meta.yaml

```diff
@@ -39,6 +39,9 @@ test:
     - q2_deblur
     - qiime2.plugins.deblur
 
+  commands:
+    - pytest --pyargs deblur
```

Comment:
@thermokarst  My _guess_ is that the CI error is due to how the tests are running due to this line; I can double check exactly what's happening but want to confirm that this change is necessary. Thank you.

Reply:
I just took a closer look at the test failures - it appears to be because deblur is not including it's test data in it's setup script:

https://github.com/biocore/deblur/blob/fa3909d24cb5da42266f22663515385a23a4b2da/setup.py#L52

Reply:
Since making a new release is a lot of work on your end, I'll just re-write this to fetch the unifrac source and run the test's that way - then it's the best of both worlds - stay tuned, and thanks for taking care of this!

Reply:
@thermokarst, thank you. I think that makes sense; anyway, just issued this to avoid this in future occasions: https://github.com/biocore/deblur/pull/216.